xcube_stac/store.py

```diff
@@ -103,36 +103,53 @@ def get_item_collection(
         return self.stac.get_item_collection(**open_params)
 
     def get_data_ids(
-        self, data_type: DataTypeLike = None, include_attrs: Container[str] = None
+        self,
+        data_type: DataTypeLike = None,
+        items: Iterable[Item] = None,
+        item_data_ids: Iterable[str] = None,
+        include_attrs: Container[str] = None,
+        **open_params
     ) -> Union[Iterator[str], Iterator[Tuple[str, Dict[str, Any]]]]:
         """Get an iterator over the data resource identifiers for the
         given type *data_type*. If *data_type* is omitted, all data
-        resource identifiers are returned.
+        resource identifiers are returned. The data resource identifiers
+        follow the following structure:
+
+            `collection_id_0/../collection_id_n/item_id/asset_id`
 
         Args:
             data_type: If given, only data identifiers
                 that are available as this type are returned. If this is None,
                 all available data identifiers are returned. Defaults to None.
-            include_attrs: A sequence of names
-                of attributes to be returned for each dataset identifier.
-                If given, the store will attempt to provide the set of
-                requested dataset attributes in addition to the data ids.
+            items: collection of items for which data IDs are desired. If None,
+                items are collected by :meth:`Stac.get_item_collection`
+                using *open_params. Defaults to None.
+            item_data_ids: data IDs corresponding to items. If None,
+                item_data_ids are collected by :meth:`Stac.get_item_data_ids`.
```

Comment:
So far, I considered the `Stac` class to be an implementation detail.
If so, you should not refer to it from public API in `StacStore`.

Reply:
I looked at other plugins, but it is not consistent. The `Cmems` class in xcube-cmems is public. The `SmosStacCatalog` class in xcube-smos is also public. The `SentinelHub` cass in xcube-sh is privat. 

So far I though, it would be nice to access the catalog of the `Stac` class, which I made private yesterday. So I will also make the `Stac` class private now.